tasks/lint.yaml

```diff
@@ -0,0 +1,8 @@
+tasks:
+  - name: run
+    description: "Lint and fix issues in the repo"
+    actions:
+      - description: "Pepr Format"
+        cmd: npx pepr format
+      - description: yaml lint
```

Comment:
still not familiar with this tool or where it is being installed. Also will this have issue with prettier?

Reply:
I dont think it would have an issue with prettier afaik prettier isnt doing anything with the yaml. That being said im not really familiar with prettier aside from having it baked in to some js projects i've worked on in the past. 

Right now that is a helper task and the answer is pip. This is the linter dubbd uses in their pipeline for yaml https://github.com/defenseunicorns/uds-package-dubbd/blob/642e948ebdc67952d00ed6dfeb8088f326d9e863/.github/workflows/ci-on-pull-request-dubbd-aws.yaml#L28

I had added these because i was getting annoyed i kept having to npx pepr format when pulling in new code even though i wasnt touching pepr code in my changes and wanted to make sure the ton of yaml was all being formatted in my dev.

I think these (or similar) would be added in a separate ticket as part of the ci process.